app/assets/stylesheets/application.scss

```diff
@@ -692,4 +692,41 @@ span.inactive {
 	font-size: 16px;
 	height: 43px;
 	width: 141px;
-}
\ No newline at end of file
+}
+
+.error-body {
```

Comment:
How much custom styling should we be doing? Are there reusable bits (like the white card on the login screen) that we can use instead?

I'm thinking something like our 404 page - https://api.simple.org/not/found - but we can leverage our existing Rails style components.

Reply:
The styling is directly stolen from the 404 page. If we weren't working async, I would've asked you about moving that styling from the 404 page into the general css, but the fact that all of the static html pages keep their styling separate gave me pause and I didn't want to hold up work until I could get a response from you. I briefly went down a path of moving the styling out and creating a common layout to use for this and the 404 page but ran into issues with the embedded link and decided I was over-complicating things.